modules/protect/blocked-login-page.php

```diff
@@ -192,7 +192,7 @@ public function render_and_die() {
 		}
 
 		if ( isset( $_GET['validate_jetpack_protect_recovery'] ) && $_GET['user_id'] ) {
-			$this->protect_die( __( 'Could not validate recovery token.', 'jetpack' ) );
+			$this->protect_die( __( 'Oops, we couldn’t validate the recovery token.', 'jetpack' ) );
```

Comment:
That `’` character may cause some confusion down the road. Should we just use `'`?